app/services/CustomsOffices.scala

```diff
@@ -16,18 +16,18 @@
 
 package services
 
+import com.github.tototoshi.csv.CSVReader
 import services.model.CustomsOffice
-import utils.FileReader
 
-import scala.util.matching.Regex
+import scala.io.Source
 
 object CustomsOffices {
 
-  private val regex: Regex = """^(\w+),"?([^"\n]+)"?$""".r
+  def all: List[CustomsOffice] = {
 
-  def all: List[CustomsOffice] =
-    FileReader("code-lists/customs-offices.csv").tail.map {
-      case regex(code: String, description: String) =>
-        CustomsOffice(code, description)
-    }.sortBy(_.description)
+    val reader = CSVReader.open(Source.fromURL(getClass.getClassLoader.getResource("code-lists/customs-offices.csv"), "UTF-8"))
+    val codes: List[Map[String, String]] = reader.allWithHeaders()
```

Comment:
👍  Didn't know about read with headers

Reply:
It was the simplest way to get the CSVReader to read and ignore the headers.  Plus getting the values as a map seems nicer than a list.

Reply:
Yeah, looks great, I used readAll or something like this, but needed to remove the headers from the file